infra/postsubmit/proto/postsubmit.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package infra.postsubmit;
+
+message Build {
+  // Name of this build. Needs to be unique among all builds, and should match
+  // the regex `[a-z][a-z_]+`.
+  string name = 1;
+
+  // Repository that this postsubmit builds code from. Should be one of:
+  // * `enfabrica/enkit`
+  // * `enfabrica/internal`
+  string repository = 2;
+
+  oneof trigger {
+    // Interval on which to run the build. This is a normal cron schedule
+    // expression (https://crontab.guru/), except that the minutes fields can
+    // (and should) be a `?` rather than fixed; this will be translated into a
+    // stable value based on the name of this build, which allows builds to not
+    // all hammer infrastructure by starting e.g. every hour on the hour.
+    string cron = 3;
+  }
+
+  // Bazel target patterns to include in the build.
+  repeated string include_patterns = 4;
```

Comment:
I feel like include and exclude are more designed for the use case of include "build/...." and exclude "build:my_heavy_test".

Maybe they should be lumped together in a message of their own and targets is a list of those messages. Makes sense to keep them tightly coupled.

Reply:
Individual tests like `my_heavy_test` should probably be excluded via tag, not this mechanism - this is more for defining which parts of the code tree belong to your team (and therefore what your team's build should cover). It's usually the case for a team to own an entire directory, but it's not unheard of for a subteam/someone else to own a subdirectory of that - this mechanism is for excluding such subdirectories.

This shouldn't be the common case - I think splitting it out adds unnecessary nesting, and the config reads nicely as-is: "include //a, //b, //c, but not //a/d, //a/e"